book/src/Example.md

```diff
@@ -128,37 +135,167 @@ Writing inside the app container should be visible in `/tmp` of the `hostpath-dr
 $ kubectl exec -ti my-csi-app /bin/sh
 / # touch /data/hello-world
 / # exit
-$ kubectl exec -ti csi-pod -c hostpath-driver /bin/sh
+$ kubectl exec -ti csi-pod-fr5bn -c hostpath /bin/sh
```

Comment:
Most of the commands can be cut-and-pasted, but not this one because of the varying pod name. How about this instead?
```
kubectl exec -ti $(kubectl get pods --selector app=csi-setup -o jsonpath='{.items[*].metadata.name}') hostpath /bin/sh
```

I wish `kubectl exec` had a --selector parameter like `kubectl logs`, but it doesn't at the moment.